lms/djangoapps/course_api/api.py

```diff
@@ -268,3 +269,65 @@ def get_course_run_url(request, course_id):
     """
     course_run_url = reverse('openedx.course_experience.course_home', args=[course_id])
     return request.build_absolute_uri(course_run_url)
+
+
+def get_course_members(course_key, include_students=True, access_roles=None, prefetch_accessroles=False, prefetch_enrollments=False):
+    """
+    Returns a User queryset that filters all users related to a course.
+    For example - Students, Teachers, Staffs etc.
+
+    Arguments:
+        course_key (CourseKey): the CourseKey for the course
+        include_students: Wether or not to include students,
```

Comment:
@shimulch Is there a reason we'd avoid retrieving students enrolled? And if so, this parameters overlaps with `access_roles`, can we use a single parameter for this with (something like `access_role=['student']` to filter students)?

Reply:
In my implementation, I've treated ``access_roles`` as valid values for ``CourseAccessRole``. ``student`` is not a valid ``access_role``. Students only have the ``CourseEnrollment`` instance but no ``CourseAccessRole`` instance. As it's an internal API, I wanted to be explicit about that.